saleor/core/middleware.py

```diff
@@ -90,14 +90,11 @@ def process_request(self, request):
         h.update(filecontent)
         hex = h.hexdigest()
 
-        secretkey = replace_last(number)
-
         if filename != hex:
             print 'hex filename not equal'
             return TemplateResponse(request, 'lockdown/form.html', {'days': 'unknown', 'machine': number})
 
-        if self.is_not_empty(filecontent):
-            jsonvalue = en.decrptcode(filecontent, secretkey)
+            jsonvalue = en.decrptcode(filecontent, number)

```

Comment:
@alexkiburu i don't know what is the argument of removing: 
```if self.is_not_empty(filecontent):``` that check if the value does not contain empty spaces. Maybe I miss something here. Note that part of the code is not under try: except block and if its empty or full of space tabs it will throw an error. Thats why we were checking it has value....
Note: NULL in database means different